### PR TITLE
Add GetTime method to Timestamp

### PR DESCRIPTION
**Pull request by 6543**
add a small handy helper func

### CODE REVIEW

File: github/timestamp.go
Comment (gmlewis) [inline github/timestamp.go:25]:
```suggestion
// GetTime returns std time.Time.
```

**Review by gmlewis — APPROVED** [review]
Thank you, @6543 .
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by yardenshoham — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @yardenshoham !
Merging.

**Pull request merged**
